resources/js/electron-builder.js

```diff
@@ -142,6 +66,27 @@ export default {
     asarUnpack: [
         'resources/**',
     ],
+    beforePack: async (context) => {
+        let arch = {
+            1: 'x64',
+            3: 'arm64'
+        }[context.arch];
+
+        // If mac is not arm, use 32bits (x86)
+        // This is how this works now according to package.json scripts,
+        // TODO: Check this. Not sure why mac x64 needs a php binary for x86? @simonhamp
```

Comment:
It's just a misnomer. I think I picked x64, but the convention elsewhere is x86... so we need to tie the two together. It's not actually a 32-bit arch, it is 64-bit, it's just notated as x86 🤷🏼

Reply:
I'll make that a separate PR once this one is ready 👍